test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+rm -rf test
+mkdir -p test
+mkdir -p test/mount_point
+mkdir -p test/gitfs_owned_repo
+mkdir -p test/origin.git
+cat > test/.gitconfig <<EOF 
+[user] 
+       name = GitFs
+       email = gitfs@gitfs.com\n
+
+EOF
+
+ROOT=`pwd`
+
+cd test/origin.git;\
+    HOME=../../test git init --bare .;\
+    cd ../../;\
```

Comment:
Those relative paths are evil. Maybe we can find a way to remove them.
For git operations we can set `GIT_WORK_TREE` and `GIT_DIR`.
Or maybe we can fix them in another pull request.